### ISSUE TITLE
Fixing Error message given during initialization: Unable to resolve dependency: user requested 'vagrant-goodhosts (= 1.0.8)'

### ISSUE THREAD

**Issue by davidwolfpaw** [Varying-Vagrant-Vagrants/VVV#2455]
I'm posting the following issue as I ran into this problem this morning, it took a bit of time to find a solution, and I want to save time if someone comes along after me and thinks to check issues as I did.

## What was The Command Used To Provision

When running any command on the existing environment, I received an error message concerning `vagrant-goodhosts`. These are some of the commands that I tried, each giving the same error message:

```
vagrant box update
vagrant box prune
vagrant reload --provision
```

## What Kind of VVV Provision Was This

This was a reprovision of an already working VVV

## Logs/What Broke

The following error was displayed:

`Error message given during initialization: Unable to resolve dependency: user requested 'vagrant-goodhosts (= 1.0.8)'`

## Possible Solution
Running `vagrant plugin install --local` which is now part of the instructions on the Readme solved this problem. After running that command and having `vagrant-goodhosts` install, I was able to successfully reprovision and run other commands. Notably uninstalling and reinstalling `vagrant-goodhosts` manually did not solve my problem.

## Your Environment
__ __ __ __
\ V\ V\ V / Varying Vagrant Vagrants
 \_/\_/\_/  v3.7.0-git::develop(e9abe7d)

Platform: platform-darwin19.2.0 shell:/bin/zsh vagrant-goodhosts shared_db_folder_disabled, VVV Path: "/Users/david/vagrant-local"
Vagrant: v2.2.16, virtualbox: v6.1.22

**Comment by Mte90** [comment]
Yes this is an issue with the vagrant plugin so doesn't involve VVV provisions etc.
Probably you are using an old vagrant-goodhosts plugin with a recent vagrant version or missing that. You can fix with `vagrant plugin update`.

**Comment by davidwolfpaw** [comment]
Correct, it is not a VVV problem, but notably updating the plugin did not help either. In my case specifically running `vagrant plugin install --local` was required to fix the issue.

There wasn't really a template for "making a public note", but the intent here is that I looked for other people experiencing the issue that I did after a recent update, and did not so I'm leaving a note for others who may be looking and wondering why `vagrant plugin update` does not work for them either. Assuming that I'm not the only one affected in this way.

**Comment by tomjn** [comment]
I don't believe you've fixed your issue at all, the globally installed copy of goodhosts is still broken. You just sidestepped it by installing a project specific copy.

As for support, the goodhosts plugin is from https://github.com/goodhosts/vagrant and the current version is 1.0.15 which is higher than the 1.0.8 you have. Also, boxes have nothing to do with vagrant plugins, and `vagrant box update` will have zero impact on existing virtual machines. Boxes are starting points for constructing new environments.

I'd suggest uninstalling and reinstalling the goodhosts plugin globally, or if you only use it with VVV, uninstalling the global copy.

**Comment by davidwolfpaw** [comment]
Thank you for the detailed information. I just confirmed that now vagrant-goodhosts is up to date at v1.0.15 locally and globally. 

I may have been making a mistake somewhere, but I tried updating, as well as uninstalling and reinstalling the plugin and restarting vagrant and it did not work for me. 

I was running an old version of vagrant-goodhosts, which is where the error came from. I did update and it is now working 👍
